modules/custom/openy_activity_finder/src/Plugin/Block/ActivityFinderBlock.php

```diff
@@ -31,12 +36,13 @@ public function build() {
       '#days' => $backend->getDaysOfWeek(),
       '#categories' => $backend->getCategoriesTopLevel(),
       '#categories_type' => $backend->getCategoriesType(),
+      '#activities' => $backend->getCategories(),
       '#locations' => $backend->getLocations(),
-      '#is_search_box_disabled' => $config->get('disable_search_box'),
       '#attached' => [
         'drupalSettings' => [
           'activityFinder' => [
-            'categories' => $backend->getCategories(),
+            'alias' => isset($alias) ? $alias : '',
```

Comment:
no need to have ternary operator when alias is defined by default, see comment for line 27 above

Reply:
done